src/axolotl/utils/vessl_.py

```diff
@@ -0,0 +1,13 @@
+"""Module for vessl utilities"""
+
+import os
+
+from axolotl.utils.dict import DictDefault
+
+
+def setup_vessl_env_vars(cfg: DictDefault):
+    # VESSL_RUN_INITIAL_CONFIG is a variable that contain path to
```

Comment:
I cannot find any references explaining this variable. Can you attach a document pointing this variable?

Reply:
I got it from container environment variables, will try to attach a screenshot